### ISSUE TITLE
There needs to be a property in the profile where the rules for the descriptive content can be provided. This would apply to the entire profile. [ID42] (5.42)

### ISSUE THREAD

**Issue by kcoyle** [w3c/dxwg#255]
This requirement is based on a use case that reflects practice primarily in libraries, but in some cases also in archives and museums.

For the last 50 years, libraries have used a specific data standard called "MARC".[1] That data standard includes a single data element to indicate the cataloging rules that were used to create the metadata. The element is called "descriptive cataloging form."[2]

18 - Descriptive cataloging form
x  - Non-ISBD
a - AACR 2
c - ISBD punctuation omitted

A new data standard called BIBFRAME[3] includes this same information as one element of administrative data[4] for the bibliographic description (aka: record). The property is called "descriptive conventions":

Property: | descriptionConventions
Label: | Description conventions
Definition: | Rules used for the descriptive content of the resource description.
Used with: | AdminMetadata
Expected Value: | DescriptionConventions

In both cases, there is one value for each "record". 

Note that the "conventions" or "rules" are in part (as Phil said) style guides, but also contain detailed instructions for making decisions. Many such decisions are not validatable, but must be left to the human cataloger to make a decision within her context, and there often is "no right answer". The current rules used by many Western libraries are over 1000 pages in length.[6] Unfortunately there isn't an open access version but I could supply examples (warning: they'd be lengthy) if anyone wishes. 

[1] http://www.loc.gov/marc/bibliographic/
[2] http://www.loc.gov/marc/bibliographic/bdleader.html
[3] http://www.loc.gov/bibframe/docs/index.html
[4] http://id.loc.gov/ontologies/bibframe.html#c_AdminMetadata
[5] http://id.loc.gov/ontologies/bibframe.html#p_descriptionConventions
[6] https://isbnsearch.org/isbn/9780838913468

**Comment by kcoyle** [comment]
Since no one has commented, here is a question:

- could there be a single solution that gives rules for both individual properties and profiles as a whole? Then we could agree this, but solve it with the other requirement

**Comment by larsgsvensson** [comment]
Given that there is a lot of copy cataloguing going on, can we really ensure that the _whole_ record is created according to only one cataloguing code... In the DNB we're looking seriously at provenance data per field and I guess the same might apply to cataloguing codes (and in the RDF world we don't have records...)

**Comment by kcoyle** [comment]
@larsgsvensson I agree that is an issue, but at the same time this is the current practice in the record-based environment. Given the general resistance to change, I suspect that this will be the practice for a while. That's why I think having both a "record-level" and a field-level capability is a good idea, and if we can have both with a single property that would be best. 

I see this as a link to or at least an identifier for the rule set. Not all such guidance rules are online.

The other thing, though, is if we consider this different from indicating generally the standard applied to a data value. If not, then the guidance rules can be included as a type of standard. I don't know if there would be cases where one would need BOTH guidance rules and a link to a formal standard. I can't think of any off-hand, but would we assume that the cardinality of standards applied to a value be non-repeatable? (0,1)?

**Comment by larsgsvensson** [comment]
@kcoyle scripsit:

> @larsgsvensson I agree that is an issue, but at the same time this is the current practice in the record-based environment. Given the general resistance to change, I suspect that this will be the practice for a while. That's why I think having both a "record-level" and a field-level capability is a good idea, and if we can have both with a single property that would be best.

+1, a single property would be easier, although I don’t quite know how that would work…

> I see this as a link to or at least an identifier for the rule set. Not all such guidance rules are online.

Indeed they are not. If they are published as books we could use urn:isbn.

> The other thing, though, is if we consider this different from indicating generally the standard applied to a data value. If not, then the guidance rules can be included as a type of standard. I don't know if there would be cases where one would need BOTH guidance rules and a link to a formal standard. I can't think of any off-hand, but would we assume that the cardinality of standards applied to a value be non-repeatable? (0,1)?

The only thing I can think of would be dates (“the value is the publication date and if that isn’t known use the manufacturing date” vs. “use an ISO format for the date”). I think the same _could_ apply to values such as floating point numbers etc. That’s why I keep saying that the cataloguing rules and the exchange formats should be kept separate…). This is sort-of-similar to the use of valueVocabulary and encodingScheme in DublinCore.

**Comment by aisaac** [comment]
I agree that the requirement is worth considering, and that it would be good if a same 'conformance' property could be used for both record level and statement level. But I believe the latter point is more a matter for the solution space and we should postpone it for now.

**Comment by jpullmann** [comment]
The requirement still confuses me. Does it mean, there might be next to 
1. a structured, machine-enforceable profile definition (schema)
2. a single property containing an opaque rule/structure as subject to human interpretation?

**Comment by kcoyle** [comment]
2. So we can add something like "human-understandable rules" if that helps. In the [Singapore Framework](http://dublincore.org/documents/singapore-framework/) diagram this is the yellow box in the upper area that reads "usage guidelines" and is defined:

Usage guidelines (optional)

The optional usage guidelines describe how to apply the application profile, how the used properties are intended to be used in the application context etc.
